test/evergreen/print_stack_trace.py

```diff
@@ -143,6 +143,13 @@ def main():
         start = output.find('execfn: ')
         # Fetch the value of execfn. This is the executable path!
         executable_path = re.search(r'\'.*?\'', output[start:]).group(0)
+        executable_path = executable_path.replace("'", "" )
+
+        # If the core dump comes from a Python test, we don't need to construct the executable path.
+        if "python" not in executable_path.lower():
+            # The executable is where the core dump is.
+            dirname = core_file_path.rsplit('/', 1)[0]
```

Comment:
what if the core is in the current dir?

Reply:
In other words, if the `core_file_path` is `dump_blah`, then the dirname would be `dump_blah` as well, no?

Reply:
My bad the `core_file_path` will be `./dump_blah`, so it shouldn't be an issue.

Reply:
Yes, tested again and it works:

From the `build` dir:
```
> ./test/format/t
program name not set: FAILED: main/197: false
Segmentation fault (core dumped)
> python3 ../test/evergreen/print_stack_trace.py
+---------------------+
| ./dump_t.22086.core |
+---------------------+
executable_path ./test/format/t
```

From the `build/test/format` dir:
```
./t
program name not set: FAILED: main/197: false
Segmentation fault (core dumped)
python3 ../../../test/evergreen/print_stack_trace.py
+---------------------+
| ./dump_t.22362.core |
+---------------------+
executable_path ./t
```

In both cases, the executable path is constructed corrected.